File: Front/src/app/interfaces/foto.ts
export interface Foto {
    id: number;
    nombreFo: string;
    data: string; // Cambia el tipo según lo que necesites para almacenar la imagen
    localId: number; // Asumiendo que localId es de tipo number
    productoId: string; // Asumiendo que productoId es de tipo number
  }

File: Back/Back/Models/Foto.cs
namespace Back.Models
{
    public class Foto
    {
        public int Id { get; set; }
        public string NombreFo { get; set; }
        public byte[] Data { get; set; }
        public string? LocalId { get; set; }
        public string? ProductoId { get; set; }

    }
}
